doc/python/px-arguments.md

```diff
@@ -198,6 +196,13 @@ fig = px.bar(wide_df, x="nation", y=["gold", "silver", "bronze"], title="Wide-Fo
 fig.show()
 ```
 
+*New in 5.16*
+
+As of version 5.16, you can also provide another type of DataFrame to the `data_frame` argument if that DataFrame supports the [Python dataframe interchange protocol](https://data-apis.org/dataframe-protocol/latest/index.html), or has a `toPandas` or `to_pandas_df` method.
+
+If you are using a type of DataFrame that doesn't have a `to_pandas`, `toPandas`, or `to_pandas_df` method, but supports the Python dataframe interchange protocol, you'll need to have Pandas version 2.0.3 or later installed.
```

Comment:
Since the dataframe interchange protocol is supposed to be preferred, I'd change this to something that encourages it, like "Even if your dataframe supports the dataframe interchange protocol, you'll need to have Pandas version 2.0.3 or later installed to use it. If an earlier version of Pandas is found we will only look for a `to_pandas`, `toPandas`, or `to_pandas_df` method."